### DISCUSSION TITLE
How to Deploy into vanilla kubernetes

### DISCUSSION THREAD

**Discussion by unixdaddy** [quay/quay#988] [Q&A]
I have tried to use the ./deploy/k8s folder in this repo, but it has old versions of api for deployment resource. it appears to be missing secrets - quay-enterprise-config-secret.yaml and there isn't clear information on how to use that folder.

Is there a cleaner method/instructions for deploying quay to kubernetes?

**Comment by eumel8** [comment]
This is completely outdated and shouldn't be used anymore. Supported deploy option is [quay-operator](https://github.com/quay/quay-operator)
